### DISCUSSION TITLE
Browser compatibility for Vite vs Webpack builds

### DISCUSSION THREAD

**Discussion by tohagan** [quasarframework/quasar#13701] [General - Components / Directives / etc] [answered]
I noticed that the "browserslist" rules in package.json were removed when migrating from Webpack to Vite. 
Does that mean that this build feature is no longer available or is it configured some place else?

**Comment by yusufkandemir** [comment, marked as answer]
https://quasar.dev/quasar-cli-vite/convert-to-quasar-cli-with-vite#4-browser-compatibility
> A Quasar CLI with Webpack project relies on `/package.json > browserslist` to specify which browsers you are targetting. That property no longer has any meaning. Projects managed by Quasar CLI with Vite work completely different and you might want to check the [Browser Compatibility](https://quasar.dev/quasar-cli-vite/browser-compatibility) page.

https://quasar.dev/quasar-cli-vite/browser-compatibility
> In order to configure the browser compatibility for your app, you will need to edit `/quasar.config.js`:
> ```js
> // quasar.config.js
> build: {
>   target: {
>     browser: [ 'es2019', 'edge88', 'firefox78', 'chrome87', 'safari13.1' ],
>     node: 'node16'
>   }
> }
> ```
